migrations/2019_11_04_165426_addclt.py

```diff
@@ -0,0 +1,19 @@
+from orator.migrations import Migration
+
+
+class Addclt(Migration):
+    def up(self):
+        """
+        Run the migrations.
+        """
+        with self.schema.table("member") as table:
+            table.boolean("clt").default(0)
```

Comment:
could this be `is_clt`? or do we use this format only on methods?

Reply:
You're right. Will update.